### PR TITLE
improve pathing

### PR DESCRIPTION
**Pull request by giogio21**

### CODE REVIEW

**Comment by arnauds5** [comment]
There is still the issue https://github.com/asfoury/projmag/issues/121 that is not solved.
If you have time, It could be better to fix it here.

File: app/src/main/java/com/sdp13epfl2021/projmag/activities/ProfilePageActivity.kt [hunk 1]
```diff
@@ -22,11 +24,22 @@ class ProfilePageActivity : AppCompatActivity() {
...
+        buttonAddCv.setOnClickListener{
+            val intent = Intent(this,CVCreationActivity::class.java)
+            startActivity(intent)
+            finish()
+        }
+        buttonSubChange.setOnClickListener{
+            val intent = Intent(this,ProjectsListActivity::class.java)
+            startActivity(intent)
+            finish()
+        }
```
Comment (PaulCoral) [inline app/src/main/java/com/sdp13epfl2021/projmag/activities/ProfilePageActivity.kt:42]: this code is duplicate maybe could be a function like this (not tested): 
```kotlin
private fun goToActivity(activityClass: ...) =....
```

File: app/src/main/java/com/sdp13epfl2021/projmag/activities/SignInActivity.kt [hunk 2]
```diff
@@ -84,10 +84,19 @@ class SignInActivity : AppCompatActivity() {
...
-                        startActivity(intent)
-                        finish()
+                        if(isNew == true){
+                            val intent = Intent(this, ProfilePageActivity::class.java)
+                            startActivity(intent)
+                            finish()
+                        } else if(isNew == false) {
+                            val intent = Intent(this, ProjectsListActivity::class.java)
+                            startActivity(intent)
+                            finish()
```
Comment (PaulCoral) [inline app/src/main/java/com/sdp13epfl2021/projmag/activities/SignInActivity.kt:96]: same as previous remark

**Review by PaulCoral — APPROVED** [review]
Not much to say, except the remark about code duplication, I let you judge.

File: app/src/main/java/com/sdp13epfl2021/projmag/activities/ProfilePageActivity.kt [hunk 3]
```diff
@@ -14,6 +14,8 @@ class ProfilePageActivity : AppCompatActivity() {
 
     lateinit var imageView: ImageView
     lateinit var button: Button
+    lateinit var buttonAddCv: Button
```
Comment (PaulCoral) [inline app/src/main/java/com/sdp13epfl2021/projmag/activities/ProfilePageActivity.kt:17]: We cant go back from CV

**Review by PaulCoral — CHANGES_REQUESTED** [review]
I didn't see this in my previous
Just add a parent in android manifest:
```xml
<activity android:name=".curriculumvitae.VideoCv"
            android:parentActivityName=".activities.ProfilePageActivity"/>
```

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -30,7 +30,7 @@
 
 
         <activity android:name=".curriculumvitae.VideoCv">
```
Comment (PaulCoral) [inline app/src/main/AndroidManifest.xml]:
```suggestion
        <activity android:name=".curriculumvitae.VideoCv"
```

**Review by PaulCoral — CHANGES_REQUESTED** [review]
Typo in manifest

**Review by PaulCoral — APPROVED** [review]

**Review by lpatel99 — APPROVED** [review]
Works well, I would just suggest adding tests even if just to press on the buttons etc. This would help to know if future changes to code make things crash for example.

**Pull request merged**